src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/CSharpFormattingOptions2.cs

```diff
@@ -274,13 +272,24 @@ private static Option2<T> CreateOption<T>(OptionGroup group, string name, T defa
         CSharpSyntaxFormattingOptions.NewLinesDefault.HasFlag(NewLinePlacement.BetweenQueryExpressionClauses))
         .WithPublicOption(PublicFeatureName, "NewLineForClausesInQuery");
 
-    static CSharpFormattingOptions2()
-    {
-        // Note that the static constructor executes after all the static field initializers for the options have executed,
-        // and each field initializer adds the created option to the following builders.
-
-        AllOptions = s_allOptionsBuilder.ToImmutable();
-    }
+    /// <summary>
+    /// Internal option -- not exposed to editorconfig tooling via <see cref="EditorConfigOptions"/>.
+    /// </summary>
+    public static readonly Option2<int> CollectionExpressionWrappingLength = new(
+        $"csharp_internal_collection_expression_wrapping_length",
```

Comment:
What actually stops it from being exposed?

Reply:
What do you mean "exposed"? Not able to use it in editorconfig file?